src/internal/components/selectable-item/styles.scss

```diff
@@ -26,12 +26,30 @@
   border-inline-end-width: 0;
   // to compensate for the loss of padding due to the removal of the left and right borders
   // and differences in default divider + selected border widths (visual refresh)
-  padding-block: styles.$option-padding-with-border-placeholder-vertical;
-  padding-inline: calc(#{styles.$control-padding-horizontal} + #{awsui.$border-item-width});
+  padding-block: calc(#{awsui.$border-item-width} - #{awsui.$border-divider-list-width});
+  padding-inline: awsui.$border-item-width;
+
+  // For custom styles and background on the option-content, the content should not clip through the selectable-item borders..
+  overflow: hidden;
```

Comment:
In which situation can the content overflow? Won't the selectable item just stretch to fit its content?

Reply:
Is it just the rounded borders that would overlap with the inner non-rounded ones? If so, it should be fine. Just making sure that actual content is not potentially clipped.